mod/item.php

```diff
@@ -175,6 +175,19 @@ function item_post(&$a) {
 	$app         = ((x($_REQUEST,'source'))      ? strip_tags($_REQUEST['source'])  : '');
 	$extid       = ((x($_REQUEST,'extid'))       ? strip_tags($_REQUEST['extid'])  : '');
 
+	// Check for multiple posts with the same message id (when the post was created via API)
+	if (($message_id != "") AND ($profile_uid != 0)) {
```

Comment:
This is not part of the PSR-2, I just saw that single quotes were used a few lines before. Like tabs, it doesn't really matter which one we choose, as long as we stick to one.